atomkraft/config/config_file.py

```diff
@@ -0,0 +1,28 @@
+import tomlkit
+
+
+class ConfigFile(object):
+    def __init__(self, path: str):
+        self.data = {}
+        self.path = path
+
+    def __enter__(self):
+        self.fd = open(self.path, "w+")
+        self.data = tomlkit.load(self.fd)
+        return self
+
+    def __exit__(self, type, value, traceback):
+        self.fd.close()
+
+    def get(self, key) -> str:
+        if key in self.data:
+            return self.data[key]
+        if key.replace("-", "_") in self.data:
+            return self.data[key.replace("-", "_")]
+        if key.replace("_", "-") in self.data:
+            return self.data[key.replace("_", "-")]
```

Comment:
Why do you have this here?

I have this in `chain` config, because it was a hack to support both `gaiad` and `junod` (they had the the same keys but with `-` and `_`)

Reply:
Right, it shouldn't be here. I removed it.